### DISCUSSION TITLE
please i want to make a request

### DISCUSSION THREAD

**Discussion by King7474** [WalletConnect/walletconnect-monorepo#2988] [Developer Support] [answered]
i have been trying for days to upgrade to the v2 wallet connect but all giudes and all not working 
please my request can the v1 still be live 
and make the v2 optional for those who wants to upgrade 

if you have help on the full guide on how to fix the v2 and make it connect to any wallet easily i will be grateful for that

**Comment by boidushya** [comment, marked as answer]
Hey, unfortunately we won't be rolling back to v1. If you need help with migration, we have extensive documentation covering both dapps and wallets over [here](https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/overview)
As far as wallets go, almost all major wallets support WalletConnect v2 and have stopped supporting v1. You can check the full list [here](https://walletconnect.com/explorer?type=wallet)
